frontend/locales/en.json

```diff
@@ -503,7 +503,8 @@
       },
       "instance": {
         "header": "About this instance",
-        "versionInfo": "Running version"
+        "versionInfo": "Running version",
+        "motdModal": "Show message of day"
```

Comment:
As the motd is titled "Information", I'd rather go with "Show instance information".

The term motd (even in the long version) is probably not known to many people and might be irritating.

Reply:
Information is a bit vague.. I changed it to "announcement"